docs/learn/networks.md

```diff
@@ -11,10 +11,10 @@ Ceramic has three public networks that you can use when building applications: M
 Mainnet is the main public network used for production deployments on Ceramic. Ceramic's mainnet nodes communicate over the dedicated `/ceramic/mainnet` libp2p topic and use [Ethereum's](../glossary#ethereum) mainnet blockchain (`EIP155:1`) for generating timestamps used in [anchor commits](../glossary#anchor-commit) for [streams](../glossary#streams). Mainnet is currently running and anyone can openly query data from streams on mainnet using the [community mainnet gateway](../tools/hosted-nodes/community-nodes#gateways), but deployments that require writing data to the network are restricted to those projects on the mainnet early launch program (ELP) waitlist. If you want to write data to mainnet, [**sign up for ELP here**](https://blog.ceramic.network/ceramic-mainnet-early-launch-program/). *Over time, we will fully open mainnet to all applications.*
 
 ### **Clay Testnet**
-Clay Testnet is a public Ceramic network used by the community for application prototyping, development, and testing purposes. Ceramic core devs also use Clay for testing official protocol release candidates. Clay nodes communicate over the dedicated `/ceramic/testnet-clay` libp2p topic and use [Ethereum's](../glossary#ethereum) Rinkeby and Ropsten testnet blockchains for generating timestamps used in [anchor commits](../glossary#anchor-commit) for [streams](../glossary#streams). **Clay is openly available for anyone to use now.**
+Clay Testnet is a public Ceramic network used by the community for application prototyping, development, and testing purposes. Ceramic core devs also use Clay for testing official protocol release candidates. Data created against the Clay nodes hosted by 3BoxLabs are not guaranteed to persist. While we aim to maintain a high level of quality on the Clay testnet that mirrors the expectations of Mainnet as closely as possible, ultimately the reliability, performance, and stability guarantees of the Clay network are lower than that of Mainnet. Because of this, **the Clay network should not be used for applications in production**.  Clay nodes communicate over the dedicated `/ceramic/testnet-clay` libp2p topic and use [Ethereum's](../glossary#ethereum) Rinkeby and Ropsten testnet blockchains for generating timestamps used in [anchor commits](../glossary#anchor-commit) for [streams](../glossary#streams). **Clay is openly available for anyone to use now.**
```

Comment:
```suggestion
Clay Testnet is a public Ceramic network used by the community for application prototyping, development, and testing purposes. Ceramic core devs also use Clay for testing official protocol release candidates. While we aim to maintain a high level of quality on the Clay testnet that mirrors the expectations of Mainnet as closely as possible, ultimately the reliability, performance, and stability guarantees of the Clay network are lower than that of Mainnet. Because of this, **the Clay network should not be used for applications in production**.  Clay nodes communicate over the dedicated `/ceramic/testnet-clay` libp2p topic and use [Ethereum's](../glossary#ethereum) Rinkeby and Ropsten testnet blockchains for generating timestamps used in [anchor commits](../glossary#anchor-commit) for [streams](../glossary#streams). **Clay is openly available for anyone to use now.**
```